addons/im_livechat/tests/test_get_mail_channel.py

```diff
@@ -52,10 +52,6 @@ def test_get_mail_channel(self):
             channel_operator_ids = [channel_operator[0] for channel_operator in channel_operators]
             self.assertTrue(all(partner_id in channel_operator_ids for partner_id in self.operators.mapped('partner_id').ids))
 
-        visitor_user_channel = self.livechat_channel._open_livechat_mail_channel('Visitor', user_id=self.visitor_user.id)
-        chat_title = '%s (%s)' % (self.visitor_user.display_name, self.visitor_user.country_id.name)
-        self.assertEqual(visitor_user_channel['correspondent_name'], chat_title, "Chat title should be correct and should contain visitor's country name")
-
```

Comment:
Why remove? Should adapt to newly returned format if appropriate?

Reply:
This part of test was making assertion on formatter that it contains already textual info for display.
That's no longer the case, now it just provides raw data.

This assertion was already out of scope of this test IMO, and that's even true with this change. We could make a test that just assert this part of `channel_info` for `im_livechat` channels, but then this would encourage to add tests for all formatters, and I really think time is too precious right now to do that.

Reply:
ok